configure.py

```diff
@@ -75,6 +75,15 @@
         dst_file="SDL2-@VERSION@.zip",
         sha256="446cf6277ff0dd4211e6dc19c1b9015210a72758f53f5034c7e4d6b60e540ecf",
     ),
+    glslang=dict(
+        # Use the legacy master-tot release until the main-tot release provides
+        # Windows binaries
```

Comment:
`master-tot` and `main-tot` are not exactly "releases"; how about:

> Use the legacy master-tot Windows build until the main-tot one is fixed, or until the glslang project provides Windows builds for their stable releases

Reply:
Changed.